docs/index.md

```diff
@@ -10,10 +10,10 @@
 
 ![Ui](images/Ui.png)
 
-**AddressBook is a desktop application for managing your contact details.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+**ContactCS** is designed for NUS Computer Science freshmen to efficiently manage and locate important contact details, including those of professors, teaching assistants, classmates, school offices, and more.
 
-* If you are interested in using AddressBook, head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
-* If you are interested about developing AddressBook, the [**Developer Guide**](DeveloperGuide.html) is a good place to start.
+* If you are interested in using ContactCS, head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
+* If you are interested about developing ContactCS, the [**Developer Guide**](DeveloperGuide.html) is a good place to start.
```

Comment:
I have also changed this file in my own PR #45, but I think your changes are much better, so I think we can just keep your changes when resolving merge conflicts, thank you!